src/cmd/build.js

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+import path from 'path';
+import {createWriteStream} from 'fs';
+import streamToPromise from 'stream-to-promise';
+
+import * as fs from '../util/promised-fs';
+import {zipDir} from '../util/zip-dir';
+import {ProgramOptions} from '../program';
+import getValidatedManifest from '../util/manifest';
+
```

Comment:
Nit + Proposal: how about introducing a new eslint rules to raise errors (or warnings) for "multiple empty lines", e.g.

```
    "no-multiple-empty-lines": [2, {"max": 1}]
``` 

or even setting "max" to 2 if we want to use two empty lines for readability from time to time.